packages/block-editor/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@faustwp/block-editor",
```

Comment:
I'm wondering if `@faustwp/block-editor` conveys the proper meaning of this project. If I stumbled upon it on NPM without prior context I'm not sure I would know the objective of the package. Thoughts?

Reply:
@blakewilson agreed it's not very clear. You have any suggestions?

Reply:
Maybe update to `@faustwp/experimental-block-editor` in order to follow suite with https://github.com/wpengine/faustjs/pull/1523. That would prevent this decision from blocking MERL-1140 and provide more time to think on this.

Reply:
> Maybe update to @faustwp/experimental-block-editor
I don't think using `experimental` here will help since we will have change the name later. Since we are using semver it's logical to keep the experimental versions tied to a semantic version.

I would change this to `@faustwp/block-editor-utils` since it contains utility helpers related to the block editor

Reply:
@theodesp I think `@faustwp/block-editor-utils` makes sense 👍